numpy/core/code_generators/cversions.txt

```diff
@@ -68,4 +68,4 @@
 # NonNull attributes removed from numpy_api.py
 # Version 16 (NumPy 1.24) No change.
 # Version 16 (NumPy 1.25) No change.
-0x00000010 = 04a7bf1e65350926a0e528798da263c0
+0x00000010 = ad404c9986484ac22c9e9cd985f2eb1e
```

Comment:
You should start a new block for 1.25, make it `0x00000011` (Version 17), note the reason, and keep the old hash for 1.23-1.24.

Reply:
looks like changing this broke the world.  do i need to change somehwere else to expect "17"?

Reply:
Look at `doc/HOWTO_RELEASE.rst`, at line 187.